armi/physics/neutronics/parameters.py

```diff
@@ -723,23 +724,23 @@ def _getNeutronicsCoreParams():
     with pDefs.createBuilder(categories=[parameters.Category.neutronics]) as pb:
         pb.defParam(
             "eigenvalues",
-            units=None,
+            units=units.UNITLESS,
             description="All available lambda-eigenvalues of reactor.",
             default=None,  # will be a list though, can't set default to mutable type.
             location=ParamLocation.AVERAGE,
         )
 
         pb.defParam(
             "kInf",
-            units=None,
+            units=units.UNITLESS,
             description="k-infinity",
             default=0.0,
             location=ParamLocation.AVERAGE,
         )
 
         pb.defParam(
             "refKeff",
-            units=None,
+            units=units.UNITLESS,
             description="Reference unperturbed keff",
```

Comment:
I love it!

My preferred approach here is to open a ticket in the downstream repo plugin. Because (a) that is probably an internal repo, and (b) work will need to be done in this repo AND that one.

Feel free to pass me the name of the repo (offline) and I can do that for you.

Reply:
Nevermind. I found it. Opening that ticket.